app/pods/search/newsletter-infos/route.js

```diff
@@ -0,0 +1,151 @@
+/* eslint-disable class-methods-use-this */
+import Route from '@ember/routing/route';
+import { isEmpty, isPresent } from '@ember/utils';
+import moment from 'moment';
+import search from 'frontend-kaleidos/utils/mu-search';
+import Snapshot from 'frontend-kaleidos/utils/snapshot';
+import { inject as service } from '@ember/service';
+
+export default class AgendaitemSearchRoute extends Route {
+  @service metrics;
+  queryParams = {
+    page: {
+      refreshModel: true,
+      as: 'pagina',
+    },
+    size: {
+      refreshModel: true,
+      as: 'aantal',
+    },
+    sort: {
+      refreshModel: true,
+      as: 'sorteer',
+    },
+  };
+
+  textSearchFields = Object.freeze([
+    'title',
+    'subTitle',
+    'data.content',
+    'richtext',
+    'themes',
```

Comment:
IMO, making the theme's part of the search input could make sense in a checkbox-like form ("Only return results having these themes"), but not in the form of plain text. Theme's being a code-list (and thus very recurrently related to news-items), will add very little in terms of accuracy of the search result.

Reply:
removed from search params (not index)